File: BungiiAutomation/Bungii.Test.Regression.Android.Integration/Pages/MenuPages/AccountPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Bungii.Test.Regression.Android.Integration.Pages
{
    class AccountPage
    {
        public AccountPage(IWebDriver driver)
        {
           PageFactory.InitElements(driver, this);
        }

        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@text='ACCOUNT']")]
        public IWebElement Header_AccountPage { get; set; }

        [FindsBy(How = How.Id, Using = "public IWebElement Account_Email { get; set; }")]
        public IWebElement Account_Name { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.widget.LinearLayout[@resource-id='com.bungii.customer:id/account_info_layout_phone']/android.widget.TextView[2]")]
        public IWebElement Account_Phone { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/account_info_textview_email")]
        public IWebElement Account_Email { get; set; }
    }
}

File: Bungii.Test.Integration/src/test/java/com/bungii/android/pages/menus/AccountPage.java
package com.bungii.android.pages.menus;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class AccountPage extends PageBase  {
    public WebElement Header_AccountPage() { return findElement("//android.widget.TextView[@text='ACCOUNT']", LocatorType.XPath); }

    public WebElement Account_Name() { return findElement("com.bungii.customer:id/account_info_textview_name", LocatorType.Id); }

    public WebElement Account_Phone() { return findElement("//android.widget.LinearLayout[@resource-id='com.bungii.customer:id/account_info_layout_phone']/android.widget.TextView[2]", LocatorType.XPath); }

    public WebElement Account_Email() { return findElement("com.bungii.customer:id/account_info_textview_email", LocatorType.Id); }
}
